services/uploads/src/updateAVDefinitions.test.ts

```diff
@@ -0,0 +1,42 @@
+import { NewClamAV } from "./clamAV"
+import { NewTestS3UploadsClient } from "./s3"
+import { updateAVDefinitions } from "./updateAVDefinitions"
+import path from 'path'
+
+describe('updateAVDefinitions', () => {
```

Comment:
🎉 🎉 🎉

Reply:
For some reason it was having its connection refused to the local s3 instance we have running, gotta chase that down, but there’s no reason we can’t test all this code locally.